src/MultiplayerMod/Multiplayer/CoreOperations/Binders/GameEventsBinder.cs

```diff
@@ -137,6 +137,9 @@ public class GameEventsBinder {
         CopySettingsEvents.Copy += args => client.Send(new CopySettings(args));
         DebugToolEvents.Modify += args => client.Send(new Modify(args));
         StampToolEvents.Stamp += args => client.Send(new Stamp(args));
+
+        MoveToLocationToolEvents.LocationSet += (navigator, movable, cell) =>
```

Comment:
Out of curiosity why this bind happens here while subscription to ChoreCreatedEvent in HostEventsBinder? While both of them are host only.